parsel/selector.py

```diff
@@ -72,6 +76,17 @@ def __getitem__(self, pos):
     def __getstate__(self):
         raise TypeError("can't pickle SelectorList objects")
 
+    def jpath(self, jpath, **kwargs):
+        """
+        Call the ``.jpath()`` method for each element in this list and return
+        their results flattened as another :class:`SelectorList`.
+
+        ``query`` is the same argument as the one in :meth:`Selector.jpath`
```

Comment:
There is no `query` parameter here (I did notice we have it wrong in `xpath` below, sorry).

Reply:
I will change it incidently